main.js

```diff
@@ -0,0 +1,117 @@
+document.getElementById("search").addEventListener("click", function () {
+  const userSearch = document.getElementById("search-query").value;
+  console.log(userSearch);
+
+  if (userSearch === "") {
+    alert("Please enter the name of a city");
+  } else {
+    fetchWeatherData(userSearch);
+  }
+
+  document.getElementById("search-query").value = "";
```

Comment:
for readability, you can extract this to a separate function like: 

```suggestion
const resetSearchInput() => {
  document.getElementById("search-query").value = "";
}
```